src/open_inwoner/configurations/models.py

```diff
@@ -316,6 +316,38 @@ class SiteConfiguration(SingletonModel):
         verbose_name=_("Plan help"),
         help_text=_("The help text for the plan page."),
     )
+
+    # search filters
+    search_filters = models.BooleanField(
+        verbose_name=_("Use filters"),
+        default=True,
+        help_text=_(
+            "Whether to show any checkboxes in order to filter the search results. If this is unchecked, none of the options below will work"
+        ),
+    )
```

Comment:
I don't think we need the first boolean as we already cover all options with the other three. A global toggle would make sense if there where a lot of options and there was need to disable them all at once.